### PR TITLE
Block Values + Multidimension Fixes

### PR DESCRIPTION
**Pull request by KrystilizeNevaDies**

### CODE REVIEW

File: code/functions.lua [hunk 1]
```diff
@@ -147,6 +147,12 @@ end
 
 
 function TeleportToIsland(a_Player, a_IslandInfo)
+    if not(a_Player:GetWorld():GetDimension() == dimOverworld) then
+        a_Player:MoveToWorld(SKYBLOCK, true)
+        SKYBLOCK:ScheduleTask(2, function ()
+        TeleportToIsland(a_Player, a_IslandInfo)
+        end)
+    end
```
Comment (Seadragon91) [inline code/functions.lua:155]: I pushed a change into master that hopefully fixes that better. Accoding to Matthias I need to send the respawn packet.

**Comment by Seadragon91** [comment]
If the change in MoveToWorld works fine on your side. I can merge this after you rebase on master.

**Comment by KrystilizeNevaDies** [comment]
The second variable in the movetoworld function is a bool, when it's true it sends the respawn packet. This works, it was a fix for Equitas and it worked in testing. Also I'm not sure what you mean by debase on master, could I get an explanation on that please?

**Comment by Seadragon91** [comment]
With rebase I mean to reapply your commit on master to include the changes in it, but it's not needed after you removed the changes in `functions,lua`

**Comment by Seadragon91** [comment]
Included the changes now locally.

**Pull request closed without merging**